### PR TITLE
added selenium version test

### PR DESCRIPTION
**Pull request by danieladamkaiser**

### CODE REVIEW

File: G1ANT.Addon.Selenium/G1ANT.Addon.Selenium.Tests/SeleniumVersionTest.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿using G1ANT.Tests;
+using NUnit.Framework;
+using System;
```
Comment (MarekKosinski) [inline G1ANT.Addon.Selenium/G1ANT.Addon.Selenium.Tests/SeleniumVersionTest.cs]: clean & sort usings

File: G1ANT.Addon.Selenium/G1ANT.Addon.Selenium.Tests/SeleniumVersionTest.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+using System.Net;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace G1ANT.Addon.Selenium.Tests
+{
+    /// <summary>
+    /// Summary description for SeleniumVersionTest
```
Comment (MarekKosinski) [inline G1ANT.Addon.Selenium/G1ANT.Addon.Selenium.Tests/SeleniumVersionTest.cs]: useless comment

File: G1ANT.Addon.Selenium/G1ANT.Addon.Selenium.Tests/SeleniumVersionTest.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    /// </summary>
+    [TestFixture]
+    [TestCategory(TestCategories.Slow)]
+    public class SeleniumVersionTest
+    {
+
+        [Test]
+        public void ValidateSeleniumVersion()
+        {
+            int result = -1;
```
Comment (MarekKosinski) [inline G1ANT.Addon.Selenium/G1ANT.Addon.Selenium.Tests/SeleniumVersionTest.cs]: ### vaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaar

change to var everywhere

Comment (danieladamkaiser) [reply G1ANT.Addon.Selenium/G1ANT.Addon.Selenium.Tests/SeleniumVersionTest.cs]: no :(

**Pull request merged**